lib/lita/handlers/jenkins.rb

```diff
@@ -12,27 +12,37 @@ class Jenkins < Handler
         'jenkins list <filter>' => 'lists Jenkins jobs'
       }
 
-      route /j(?:enkins)? b(?:uild)? ([\w\.\-_ ]+)/i, :jenkins_build, command: true, help: {
+      route /j(?:enkins)? b(?:uild)? ([\w\.\-_ ]+)(, (.+))?/i, :jenkins_build, command: true, help: {
         'jenkins b(uild) <job_id or job_name>' => 'builds the job specified by ID or name. List jobs to get ID.'
       }
 
-      def jenkins_build(response)
-        job = find_job(response.matches.last.last)
+      def jenkins_build(response, empty_params = false)
```

Comment:
@daniely I'm not certain I understand fully.
The intent of the method argument `empty_params` is to control which constructed path is selected from the `job_build_url` method when receiving a 400 response due to the Jenkins build not being passed any parameters, and issuing a rebuild of the job.

It's kind of recursive in that nature, where `jenkins_build` calls `jenkins_build` again, with passing `empty_params` as true.

I tried changing the method to: `def jenkins_build(response, empty_params: false)` and this seems to work. But I'm uncertain how that's any different?